components/NavBar.tsx

```diff
@@ -84,20 +84,23 @@ function NavBar() {
                     GET STARTED
                   </Link>
                 </MenuItem>
-                {/* <MenuItem>
+                <MenuItem>
                   <Link
                     as={NextLink}
-                    href="/lessons"
+                    href="https://airtable.com/appDMMIARfSeiovpk/shrZExypPetXEx6Ox"
```

Comment:
external links should use the normal anchor tag `<a>` 

can you change it?? thanks !